src/nifreeze/model/dmri.py

```diff
@@ -178,8 +175,8 @@ def __init__(
         self,
         dataset: DWI,
         stat: str = "median",
-        th_low: float = 100.0,
-        th_high: float = 100.0,
+        th_low: float = BVAL_THRESHOLD,
+        th_high: float = BVAL_THRESHOLD,
```

Comment:
Apologies, this was a really bad name for the parameter:

```suggestion
        atol_low: float = BVAL_ATOL,
        atol_high: float = BVAL_ATOL,
```